key.go

```diff
@@ -139,6 +139,14 @@ const (
 	keyUS  KeyType = 31  // unit separator
 	keySP  KeyType = 32  // space
 	keyDEL KeyType = 127 // delete. on most systems this is mapped to backspace, I hear
+	KeyCU  KeyType = 200 // ctrl+up
+	KeyCD  KeyType = 201 // ctrl+down
+	KeyCR  KeyType = 202 // ctrl+right
+	KeyCL  KeyType = 203 // ctrl+left
+	KeySU  KeyType = 204 // shift+up
+	KeySD  KeyType = 205 // shift+down
+	KeySR  KeyType = 206 // shift+right
+	KeySL  KeyType = 207 // shift+left
```

Comment:
You can actually drop these altogether. This section is actually for very specific, predefined ANSI control key sequences.

Reply:
Done